api/src/pcapi/core/finance/api.py

```diff
@@ -2804,3 +2818,7 @@ def update_bank_account_venues_links(
             history_models.ActionHistory,
             action_history_bulk_insert_mapping,
         )
+
+
+def is_cashflow_script_running() -> bool:
```

Comment:
Je ne suis pas fan du nom. Il y a 2 opérations qui traitent des cashflows : la génération des cashflows (qui les créé) et la génération des justificatifs de remboursement (qui change leur statut). Suggestion de renommage : 

```suggestion
def are_cashflows_being_generated() -> bool:
```